src/app/doctor/doctor.cpp

```diff
@@ -28,3 +28,20 @@ void Doctor::createDoctor(const std::string& name, const std::string& surname, c
     Doctor doctor{name, surname, pesel};
     Clinic::appendDoctor(std::make_shared<Doctor>(doctor));
 }
+
+std::shared_ptr<Doctor>& Doctor::getDoctor(const std::string& name, const std::string& surname)
+{
+    auto doctors = Clinic::getDoctors();
+    auto exact_doctor = std::find_if(doctors.begin(), doctors.end(), [&](const auto& doctor) {
+        return (doctor->getName() == name) && (doctor->getSurname() == surname);
+    });
+
+    if (exact_doctor != doctors.end())
+    {
+        return *exact_doctor;
+    }
+    else
+    {
+        throw std::runtime_error("Doctor not found.");
+    }
+}
```

Comment:
What if we'll find few doctors with the same name and surname? Please fix this so it'll return some container with pointers to these objects.